examples/src/fibonacci/utils.rs

```diff
@@ -5,9 +5,9 @@
 
 use winterfell::math::{fields::f128::BaseElement, FieldElement};
 
-pub fn compute_fib_term(n: usize) -> BaseElement {
-    let mut t0 = BaseElement::ONE;
-    let mut t1 = BaseElement::ONE;
+pub fn compute_fib_term<T: FieldElement>(n: usize) -> T {
+    let mut t0 = T::ONE;
+    let mut t1 = T::ONE;
```

Comment:
minor nit: we usually use `E` for field elements.

Reply:
done, thanks